README.md

```diff
@@ -334,7 +334,7 @@ Internal Revenue Service; Bloomberg BNA; American Petroleum Institute; Alcohol a
 
 ### <a id="table12"></a>Table 12. State Individual Income Tax Rates
```

Comment:
These edits are to Table 12. This is the second PR with table numbers different in the description from what's in `README.md`. Have table numbers changed? They should be edited in `README.md` to reflect those changes if so.